docs/quick_start.md

```diff
@@ -33,6 +33,51 @@ conda create --name deepreg python=3.7 tensorflow-gpu=2.2 # use conda to handle
 conda activate deepreg
 ```
 
+- Regularly tested on Ubuntu/Debian.
+
+#### ** MacOS **
+
+With CPU only
+
+```terminal
+conda create --name deepreg python=3.7 tensorflow=2.2
+conda activate deepreg # activate the environment
+```
+
+With GPU
+
+```terminal
+conda create --name deepreg python=3.7 tensorflow-gpu=2.2 # use conda to handle nvidia related packages
+conda activate deepreg
+```
+
+- Regularly tested on Ubuntu/Debian. To check tests pass locally, run:
+
+```terminal
+conda activate deepreg
+pytest --cov-report=term --cov=deepreg test/unit/
```

Comment:
if i understand correctly, we want to let users to test their installations by running unit test,
then i recommend to move this part outside of the tab,
as this is not only for mac ;)

Reply:
because, it seems that the linux/mac/win tabs are not synced